scripts/nfs-roles-management/internal/synchronization/synchronization.py

```diff
@@ -122,6 +127,10 @@ def synchronize_user(self, user, use_symlinks=False):
                     if storage_user.username.strip().lower() == user.strip().lower():
                         syncing_storages[storage] = storage_user_mask
 
+            if not details or details.blocked:
+                logging.warning('Skipping user {} storages linking because the user is blocked...'.format(user))
```

Comment:
How about to add `details` dict to log message or some how mark in a log message what was the case here, user was blocked or details isn't provided?